sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.aest.controller.updateProfileInformation.e2e-spec.ts

```diff
@@ -0,0 +1,181 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { IdentityProviders } from "@sims/sims-db";
+
+describe("StudentAESTController(e2e)-getUploadedFile", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should allow the student profile update when the student is a Basic BCeID user.", async () => {
+    // Arrange
+    const student = await saveFakeStudent(db.dataSource);
+    student.user.identityProviderType = IdentityProviders.BCeIDBasic;
+    await db.student.save(student);
+    const studentProfileUpdateInfo = {
+      givenNames: "SCOOBY",
+      lastName: "DOOBY-DOO",
+      birthdate: "1980-03-24",
+      email: "dummy@some.domain",
```

Comment:
As a suggestion, to avoid being creative while generating faking names we have the `faker` lib for these situations.

Reply:
Will update it.